flink-filesystems/flink-azure-fs-hadoop/pom.xml

```diff
@@ -47,6 +47,14 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
+		<!-- ArchUit test dependencies -->
```

Comment:
typo `ArchUnit`. Same issue in all poms.

Reply:
good catch! Thanks!